qutip/about.py

```diff
@@ -61,6 +63,19 @@ def about():
     qutip_install_path = os.path.dirname(inspect.getsourcefile(qutip))
     print("Installation path:  %s" % qutip_install_path)
 
+    # family packages
+    entrypoints = pkg_resources.iter_entry_points('qutip.about')
+    for ep in entrypoints:
+        about_func = ep.load()
+        try:
+            title, lines = about_func()
+        except Exception as exc:
+            title, lines = ep.name, [str(exc)]
+        print()
+        print(title)
+        for line in lines:
+            print(line)
```

Comment:
```suggestion
            print(line)
        print()
```
Add an empty line incase there are other family packages